codebundles/mongodb-health-gcp-promql/sli.robot

```diff
@@ -0,0 +1,193 @@
+*** Settings ***
+Metadata          Author    Shea Stewart            
+Documentation     Uses promql on the Ops Suite API to determine the health of a MongoDB database instance
+...               and pushes the result as an SLI metric. Produces a 1 for a healthy resource, or 0 for an unhealthy resource. 
+Force Tags        GCP    OpsSuite    PromQL    MongoDB 
+Library           RW.GCP.OpsSuite
+Library           RW.Core
+Library           RW.Utils
+Library           RW.Prometheus
+Library           String
+Library           Collections
+Suite Setup       Suite Initialization
+
+*** Tasks ***
+Get Access Token
```

Comment:
Interesting that you made this its own task instead of in the initialization. Why's that?